Strategy11/formidable-modal
# Implement AJAX Submission Notification for Formidable Modal

This PR introduces a user notification for enabling AJAX submission within Formidable Modals to enhance the user experience.

## Issue URL:
[Issue #17](https://github.com/Strategy11/formidable-modal/issues/17)

## QA URL:

## Testing Instructions:
Refer to the [Bootstrap Modal Popup knowledge base](https://formidableforms.com/knowledgebase/bootstrap-modal/), and then add a Formidable Modal inside the WordPress Block Editor. Verify that there is an alert message displayed. This message should inform you about the AJAX submission feature for enhanced user experience.

## Output Comparisons:
### Before:
![CleanShot 2024-07-16 at 21 38 02@2x](https://github.com/user-attachments/assets/7f3bade2-bbe3-429c-957b-8be4e01f9e51)

### After
![CleanShot 2024-07-16 at 21 36 32@2x](https://github.com/user-attachments/assets/b47dee4f-d365-4ecc-9e80-a8dade13c079)
